internal/adapter/employee.go

```diff
@@ -30,6 +30,7 @@ type EmployeeProfile struct {
 	DepartmentId types.Int64  `tfsdk:"department_id"`
 	Team         types.String `tfsdk:"team"`
 	TeamId       types.Int64  `tfsdk:"team_id"`
+	Office		 types.String `tfsdk:"office"`
```

Comment:
You will also have to assign the `Office` attribute in the [`convertProfile` function](https://github.com/nicoangelo/terraform-provider-personio/blob/8f353e5e0235e2563c96b74571d0ebe40170af14/internal/adapter/employee.go#L163)